packages/test-domain/src/domain/Cart.ts

```diff
@@ -35,7 +35,7 @@ export namespace Fold {
   export type State = { items: ItemInfo[] }
   export const toSnapshot = (s: State): Events.Compaction.State => ({ items: s.items })
   export const ofSnapshot = (s: Events.Compaction.State) => ({ items: s.items })
```

Comment:
ugh I need to unify the module snapshot and compaction terminology in the upstream
compaction was from Eirik's original terms in SaveForLater that he wrote as a bespoke thing, which I generalized to the loading backwards to an origin scheme

Reply:
Will do something about it after merge